src/utils.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+
+export const PLACEHOLDER_REGEX = /xxx(\S)xxx/gm
```

Comment:
@threepointone can you chime in on this. I'm not 100% and don't want to hold this up due to a hunch.